pymobiledevice3/cli/developer.py

```diff
@@ -841,6 +841,15 @@ def accessibility_settings_set(service_provider: LockdownClient, setting, value)
     service.set_setting(setting, eval(value))
     OSUTILS.wait_return()
 
+@accessibility_settings.command('reset', cls=Command)
+def accessibility_settings_set(service_provider: LockdownClient):
+    """
+    reset accessibility settings to default
+
+    in order to list all available use the "show" command
```

Comment:
remove this line as this isn't relevant to this command as an argument

Reply:
Done in [this](https://github.com/doronz88/pymobiledevice3/pull/1297/commits/dea8673f662ca1b0bda58293dfa84004358aa071) commit.